compiler/damlc/tests/src/DA/Test/DamlDoc.hs

```diff
@@ -8,9 +8,12 @@ import qualified DA.Daml.Doc.Tests as Damldoc
 import qualified DA.Daml.Doc.Render.Tests as Render
 
 import qualified Test.Tasty.Extended as Tasty
+import System.Environment.Blank
 
 main :: IO ()
-main = Tasty.deterministicMain =<< allTests
+main = do
+    setEnv "TASTY_NUM_THREADS" "1" True
```

Comment:
What changed that prevents us from running tests in parallel?

Reply:
Nothing, but they are much harder to debug.